packages/sanity/src/core/releases/tool/components/ReleaseMenuButton/__tests__/ReleaseMenuButton.test.tsx

```diff
@@ -1,21 +1,23 @@
-import {fireEvent, render, screen} from '@testing-library/react'
+import {fireEvent, render, screen, waitFor} from '@testing-library/react'
 import {act} from 'react'
 import {beforeEach, describe, expect, test, vi} from 'vitest'
 
 import {createTestProvider} from '../../../../../../../test/testUtils/TestProvider'
+import {activeScheduledRelease} from '../../../../__fixtures__/release.fixture'
 import {releasesUsEnglishLocaleBundle} from '../../../../i18n'
 import {type ReleaseDocument} from '../../../../index'
+import {useReleaseOperationsMock} from '../../../../store/__tests__/__mocks/useReleaseOperations.mock'
 import {useReleaseOperations} from '../../../../store/useReleaseOperations'
 import {ReleaseMenuButton, type ReleaseMenuButtonProps} from '../ReleaseMenuButton'
 
-vi.mock('sanity', () => ({
-  SANITY_VERSION: '0.0.0',
-  useTranslation: vi.fn().mockReturnValue({t: vi.fn()}),
+vi.mock('../../../../store/useReleaseOperations', () => ({
```

Comment:
Tried in vain to simplify the way this could be mocked, but a mixture of issues an nuance with how `vi.mock` is hoisted and executed mean we still need these mock statements explicitly in our tests rather than in generalised mock files